Tasks/AzureCLIV2/task.json

```diff
@@ -0,0 +1,200 @@
+    "groups": [
+        {
+            "name": "advanced",
+            "displayName": "Advanced",
+            "isExpanded": true
+        }
+    ],
+    "inputs": [
+        {
+            "name": "connectedServiceNameARM",
+            "aliases": [
+                "azureSubscription"
+            ],
+            "type": "connectedService:AzureRM",
+            "label": "Azure temp subscription",
+            "required": true,
+            "helpMarkDown": "Select an Azure resource manager subscription for the deployment",
+            "properties": {
+                "EndpointFilterRule": "ScopeLevel != ManagementGroup"
+            }
+        },
+        {
+            "name": "scriptType",
+            "type": "pickList",
+            "label": "Script Type",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Type of script: Bat/Shell/PS/PS core script",
+            "options": {
+                "batch": "Batch script",
+                "bash": "Shell script",
+                "ps": "Powershell 5 script",
+                "pscore": "Powershell Core 6 script"
+            }
+        },
+        {
+            "name": "scriptLocation",
+            "type": "pickList",
+            "label": "Script Location",
+            "defaultValue": "scriptPath",
+            "required": true,
+            "helpMarkDown": "Type of script: File path or Inline script",
+            "options": {
+                "inlineScript": "Inline script",
+                "scriptPath": "Script path"
+            }
+        },
+        {
+            "name": "scriptPath",
+            "type": "filePath",
+            "label": "Script Path",
+            "defaultValue": "",
+            "required": true,
+            "visibleRule": "scriptLocation = scriptPath",
+            "helpMarkDown": "Fully qualified path of the script(.bat or .cmd when using Windows based agent and .sh when using linux based agent) or a path relative to the the default working directory"
+        },
+        {
+            "name": "inlineScript",
+            "type": "multiLine",
+            "label": "Inline Script",
+            "defaultValue": "",
+            "required": true,
+            "visibleRule": "scriptLocation = inlineScript",
+            "helpMarkDown": "You can write your scripts inline here. When using Windows agent, use batch scripting whereas use shell scripting when using Linux based agents. For batch files use the prefix \"call\" before every azure command. You can also pass predefined and custom variables to this script using arguments \n\n example for shell: \naz --version \naz account show \n\n example for batch:\ncall  az --version \ncall az account show",
+            "properties": {
+                "resizable": "true",
+                "rows": "10",
+                "maxLength": "5000"
+            }
+        },
+        
+
+        {
+            "name": "powerShellErrorActionPreference",
```

Comment:
do you want to move this input and `powerShellIgnoreLASTEXITCODE` input to a separate group for PowerShell which is visible only when ps or pscore is selected for scripttype

Reply:
as discussed it might be okay to keep it as it is

Reply:
just see if we need to move it after Arguments inputs